docs/cli/bun-install.md

```diff
@@ -95,7 +95,7 @@ disableManifest = false
 [install.lockfile]
 
 # Print a yarn v1 lockfile
-# Note: it does not load the lockfile, it just converts bun.lockb into a yarn.lock
+# Note: it does not load the lockfile, it just converts bun.lock into a yarn.lock
```

Comment:
this change may be inaccurate but the rest lgtm

Reply:
```
bun init -y
bun install --yarn
ls
  bun.lock
  yarn.lock
```
Seems accurate

Reply:
this bunfig option only makes `--yarn` default to on? ah my mistake then. assumed it might interact with `bun bun.lockb` in some way